miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java

```diff
@@ -159,69 +163,55 @@ private void initializeNamingSchemes(XmlWebApplicationContext context, Map<Strin
     log.info("Replacing default namingScheme with " + scheme.getClass().getSimpleName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Format specifiers should be used instead of string concatenation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3457)